static/js/components/LRDrawerPaginationControls.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React from "react"
+
+type Props = {
+  page: number,
+  begin: number,
+  setPage: Function,
+  count: number,
+  end: number
+}
+
+export default function LRDrawerPaginationControls(props: Props) {
+  const { page, begin, setPage, count, end } = props
+
+  return (
+    <div className="pagination-nav">
```

Comment:
I couldn't find the mock that contains these previous / next buttons, but see Abdul's comments about the width.